js/apps/system/_admin/aardvark/APP/react/src/components/split/Split.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import ReactSplit, {
+  ChildrenPattern,
+  ComponentPattern,
+  RenderPattern,
+  SplitGridProps
+} from "react-split-grid";
+
+export const Split = (
+  props: SplitGridProps &
+    (ChildrenPattern | RenderPattern | ComponentPattern) & {
+      localStorageKey: string;
```

Comment:
How about we rename this to `accesskey` ?

Reply:
used `storageKey` because we have this eslint rule, `accessKey` is keyword https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-access-key.md